model/event/admin_event.go

```diff
@@ -138,3 +138,26 @@ func convertRaw(in rawAdminEventData) (*AdminEventData, error) {
 
 	return &out, nil
 }
+
+// RevertConfig reverts one config section to the before state of the specified GUID in the event log
+func RevertConfig(guid string, user string) error {
+	events, err := FindAdmin(ByGuid(guid))
+	if err != nil {
+		return errors.Wrap(err, "problem finding events")
+	}
+	if len(events) == 0 {
+		return fmt.Errorf("unable to find event with GUID %s", guid)
```

Comment:
`errors.Errorf`

Reply:
Both are nearly the same thing, but it looks like errors.Errorf has a stack (which isn't needed here)